apps/admin-portal/package.json

```diff
@@ -29,7 +29,9 @@
         "@wso2is/validation": "^1.0.2"
     },
     "devDependencies": {
-        "run-script-os": "^1.0.7"
+        "fork-ts-checker-webpack-plugin": "^4.0.3",
+        "run-script-os": "^1.0.7",
+        "thread-loader": "^2.1.3"
```

Comment:
This as well. As these will be use commonly in other packages.

Reply:
Resolved with commit `Fix peer dependencies` commit.